osu.Game/Rulesets/Scoring/ScoreProcessor.cs

```diff
@@ -296,7 +301,7 @@ private void updateScore()
                     TotalScore.Value = max_score * (base_portion * baseScore / maxBaseScore + combo_portion * HighestCombo / maxHighestCombo) + bonusScore;
                     break;
                 case ScoringMode.Exponential:
-                    TotalScore.Value = (baseScore + bonusScore) * Math.Log(HighestCombo + 1, 2);
+                    TotalScore.Value = (baseScore + bonusScore) * Math.Max(0, HighestCombo - 1) / 25f;
```

Comment:
Maybe add a comment explaining why are you dividing by 25 and removing the exponential part of the equation?

Reply:
See this PR/commit's description.